src/styles/ui-components.scss

```diff
@@ -8,34 +8,32 @@
   overflow: hidden;
 }
 
-
 /* Begin Patternfly Toolbar overrides */
 
-.toolbar-pf .form-group .btn,
-.toolbar-pf .form-group .btn-group { /* overrides _toolbar.scss to account for intermediate Angular markup */
-  margin-left: 5px;
-}
 
-.form-group.miq-toolbar-group { /* styling for "Custom Html Toolbar Menu" example (used on classic-ui topology screen) */
-  display: inline-block;
-  .miq-custom-html {
-    form {
-      display: inline-block;
-    }
-    .form-group {
-      margin-bottom: 0 !important;
-      border-right: 0;
-      &:last-child {
-        padding-right: 0px
+/* overrides _toolbar.scss to account for intermediate Angular markup and adjusted the margins to properly style wrapped form-groups */
+
+.miq-toolbar-actions {
+  margin-top: -10px;
+  .form-group:first-of-type {
+    padding-left: 0;
+  }
+  .form-group {
+    margin-top: 10px;
+    padding-left: 20px;
+    padding-right: 15px;
+    white-space: nowrap;
+    .btn {
```

Comment:
Looks like custom toolbars are inconsistent, any toolbar group with button in it has spacing 20px on right, however if toolbar group ends with anything else (label, input...), it has only 15px.
* `label`
![screenshot from 2017-09-26 15-48-19](https://user-images.githubusercontent.com/3439771/30864094-fe65c356-a2d2-11e7-81c8-5262c706d03a.png)
* `search-pf-input-group` class or `input`
![screenshot from 2017-09-26 15-54-49](https://user-images.githubusercontent.com/3439771/30864118-0e58c592-a2d3-11e7-87fe-a4adab81055c.png)